backend/routes/scores/getDailyChallengeScores.ts

```diff
@@ -3,14 +3,14 @@ import { NextApiRequest, NextApiResponse } from 'next'
 import queryTopScores from '@backend/queries/topScores'
 import { getUserId, throwError, todayEnd, todayStart } from '@backend/utils'
 
-const getScoresHelper = async (userId: string | undefined, query: any, res: NextApiResponse) => {
-  const data = await queryTopScores(query, 5)
+const getScoresHelper = async (userId: string | undefined, query: any, res: NextApiResponse, limit: number | undefined) => {
+  const data = await queryTopScores(query, limit || 5)
```

Comment:
Suggestion: `limit ? Math.min(limit, 100) : 5

If your use case requires a higher limit then don't worry about it, might be a good idea to just give the limit some cap

Reply:
I have set a max limit of 200 if that's okay with you. This should be high enough for the current size of instances.